<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: bisque;</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="stock_tab">
       <attribute name="title">
        <string>Stock</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="update_inventory_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(224, 122, 95);
color:  rgb(0, 43, 91);
border-radius: 10px;
}

QPushButton::hover:!pressed {
	
	background-color: rgb(199, 107, 84);
	color: rgb(0, 43, 91);

}</string>
          </property>
          <property name="text">
           <string>Add Item in Inventory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="reload_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(224, 122, 95);
color:  rgb(0, 43, 91);
border-radius: 10px;
}

QPushButton::hover:!pressed {
	
	background-color: rgb(199, 107, 84);
	color: rgb(0, 43, 91);

}</string>
          </property>
          <property name="text">
           <string>Reload Table</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="stockTable">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {
	background-color: #002B5B;
    color: rgb(143, 227, 207);
	font: 12pt &quot;Kalpurush&quot;;
}



QTableWidget {
	background-color: #256D85;
	gridline-color: rgb(255, 255, 255);
	color: #fff;
}</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>180</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Product</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Available Stock</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Purchase Possibility</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>Apple</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>20%</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="best_products_tab">
       <attribute name="title">
        <string>Best Products</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Best Sold Product</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>271</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="product_count_edit">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(224, 122, 95);
padding: 15px;
font: 15pt &quot;Kalpurush&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>eg. 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="best_product_table">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 43, 91)</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>180</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Product Combinition</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Confidence</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="transaction_tab">
       <attribute name="title">
        <string>Transaction</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>92</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>123</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLineEdit" name="product_names_edit">
          <property name="minimumSize">
           <size>
            <width>671</width>
            <height>101</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(224, 122, 95);
padding: 15px;
font: 20pt &quot;Kalpurush&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Product Names (eg. Mango, Juice, )</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>123</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLineEdit" name="product_quant_edit">
          <property name="minimumSize">
           <size>
            <width>671</width>
            <height>101</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>671</width>
            <height>101</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(224, 122, 95);
padding: 15px;
font: 20pt &quot;Kalpurush&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Product Amount (eg. 10,20)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>128</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>56</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0" colspan="2">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>308</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="add_transaction_btn">
          <property name="minimumSize">
           <size>
            <width>301</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>301</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(0, 43, 91);
color:  white;
border-radius: 10px;
}

QPushButton::hover:!pressed {
	
	background-color:  rgb(0, 29, 61);
	color: white;

}</string>
          </property>
          <property name="text">
           <string>Add Transaction</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" colspan="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>313</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>72</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
